pyembed/build.rs

```diff
@@ -135,12 +136,23 @@ fn main() {
 
     println!("cargo:rustc-cfg=library_mode=\"{}\"", library_mode);
 
+    let pyo3_config_path = env::var("DEP_PYTHON_INTERPRETER_CONFIG_PATH")
+        .expect("DEP_PYTHON_INTERPRETER_CONFIG_PATH not found; this should have been set by the build system via the pyo3 crate");
+
+    let interpreter_config = PyO3InterpreterConfig::from_reader(
+        &std::fs::File::open(&pyo3_config_path)
+            .expect("could not open pyo3 config file for reading"),
+    )
+    .expect("failed to parse PyO3 configuration file");
```

Comment:
I'd be really curious to know if things work if you try this instead:

```suggestion
    let interpreter_config = pyo3_build_config::get();
```

... my expectation is that it should work fine; you use the `PYO3_CONFIG_FILE` env var so as long as that's present throughout the whole compilation, I expect this API should just work without needing any `DEP_` variable.